.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+### All submissions
+
+* [ ] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
+* [ ] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change
+
+### Adding/updating Application X Support
+
+* [ ] This PR is only for one application
+* [ ] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
+* [ ] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
+* [ ] Syncing does not break the application
+* [ ] Syncing does not break any syncing functionality internal to the application
```

Comment:
```suggestion
* [ ] Syncing does not compete with any syncing functionality internal to the application
```

Reply:
> This is great, my only comment is a clarification about the relationship between Mackup and application that offer a syncing functionality. I always thought Mackup should not compete with those, especially to be fair with the developers who implemented this functionality.
>
> Do you think it's fair?

That's more than fair enough. Thanks for coming back to this PR @lra, it's greatly appreciated. I'll now try and figure out how to sort out all the commits...